examples/basic_usage.py

```
@@ -0,0 +1,133 @@
+    Evaluate a mathematical expression.
+    
+    Args:
+        expression: The mathematical expression to evaluate.
+        
+    Returns:
+        The result of the evaluation.
+    """
+    # In a real implementation, use a safe evaluation
+    print(f"Calculating: {expression}")
+    try:
+        return eval(expression, {"__builtins__": {}}, {})
```

Ruff (0.13.1)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)